sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/CBDE/MlirTestUtilities.cs

```diff
@@ -71,7 +69,12 @@ public static void ValidateWithReference(string code, string expected, string te
             var expectedLines = trimmedExpected.Split('\n');
             var actualLines = trimmedActual.Split('\n');
             var maxLines = Math.Min(expectedLines.Length, actualLines.Length);
-            for (int i = 0; i < maxLines; ++i)
+            Console.WriteLine("Expected:");
+            Console.WriteLine(trimmedExpected);
+            Console.WriteLine();
+            Console.WriteLine("Actual:");
+            Console.WriteLine(trimmedActual);
```

Comment:
I think you forgot to remove these :)

Reply:
No, it's intentional improvement. Just like the dump of UCFGs and the "Open additional output for this result" in tests.
The utility only reports difference of the first unexpected line. That's a nightmare to debug/diagnose.